bundles/org.openhab.binding.fenecon/README.md

```diff
@@ -0,0 +1,146 @@
+# FENECON Binding
+
+The FENECON Binding integrates the [FENECON energy storage system](https://fenecon.de/) device into the openHAB system via [REST-API](https://docs.fenecon.de/_/de/fems/fems-app/OEM_App_REST_JSON.html).
+
+With the binding, it is possible to request status information from FENECON Home to allow you home automation decisions based on the current energy management.
+
+This makes it possible, for example, to switch on other consumers such as the dishwasher or washing machine in the case of power overproduction.
+
+## Supported Things
+
+Only one Thing is supported: The `fenecon` device.
```

Comment:
Is a `Thing` with the same name as the binding future proof?  Maybe better to change to `device` or something else. In the future more `Things` might be added.

Reply:
Good point, I changed it to `rest-api`.

Reply:
But what if the api adds more devices that need some configuration? The current line up exists around home 10 devices (if I’m correct, just had a Quick look at their website) so maybe change it to `home-device` ?

Reply:
Yes, `home-device` is the better choice, I have changed it.